pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -264,6 +264,7 @@ var _ = Describe("Pod Network", func() {
 
 	Context("on successful setup", func() {
 		It("should define a new VIF bind to a bridge", func() {
+			mockNetwork.EXPECT().IsIpv4Primary().Return(true).AnyTimes()
```

Comment:
Please specify the exact amount of occurrences. Same for all the places in the file.

Reply:
np, but why, so it will be more sensitive ?
(really ask, because i had advantage to the both options)

Reply:
Yes, so it will be more sensitive. One of the purposes of the unit tests IMO is to see that what think is what actually happen. There is no reason `IsIpv4Primary` to be invoked more than once. So let's verify it.